discovery/eureka_test.go

```diff
@@ -0,0 +1,119 @@
+package discovery
+
+import (
+	"gopkg.in/ory-am/dockertest.v3"
+	dc "github.com/fsouza/go-dockerclient"
```

Comment:
why does "dc" mean?

Reply:
dockerclient. I think I picked as the reference dockertest tests ;) ( https://github.com/ory/dockertest/blob/v3/dockertest_test.go#L96 ).  If they are importing with this shortname, probably there's a reason (clash?)

Reply:
ok, it's the shortname